### PR TITLE
[DOCS] rephrased impulse

### PR DESCRIPTION
**Pull request by toger5**
- improove coordinate system explanation
 - define usecase
@wmww can u review?

### CODE REVIEW

File: doc/classes/PhysicsDirectBodyState.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
 			<argument index="1" name="j" type="Vector3">
 			</argument>
 			<description>
-				Apply a positioned impulse (which will be affected by the body mass and shape). This is the equivalent of hitting a billiard ball with a cue: a force that is applied once, and only once. Both the impulse and the position are in global coordinates, and the position is relative to the object's origin.
+				Applies a positioned impulse to the body. An impulse is time independant! Applying an impulse every frame would result in a framerate dependant force. Thats why it should only be used, when simulating one-time impacts. The position uses the global coordinate system (the global rotation) but it is relative to the object's origin.
```
Comment (cbscribe) [inline doc/classes/PhysicsDirectBodyState.xml]: "independent" and "dependent" are spelled wrong.
 > Thats why it should only be used, when simulating one-time impacts

"Thats (sic) why" -> "For this reason,"
Remove the comma after "used"

**Review by cbscribe — CHANGES_REQUESTED** [review]
Spelling and grammar corrections

**Comment by wmww** [comment]
@toger5 Thanks for PRing this! I sure wasn't going to.

How about this:

> Applies a positioned impulse to the body. The force applied is not dependent on physics FPS, so instead of applying an impulse each frame, you should probably use [method add_force] instead. The position uses the rotation of the global coordinate system, but is centered at the object's origin.

**Comment by toger5** [comment]
@wmww I kind of want to mention that it is supposed to be used for one time impacts.
ill change that part to ur version for sure though:
> The position uses the rotation of the global coordinate system, but is centered at the object's origin.

**Comment by toger5** [comment]
updated.
any further changes requested?

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**